src/saveload/saveload.h

```diff
@@ -385,6 +385,8 @@ enum SaveLoadVersion : uint16_t {
 	SLV_ROAD_WAYPOINTS,                     ///< 338  PR#12572 Road waypoints
 	SLV_ADD_DEPOTS_TO_HANGARS,              ///< XXX  PR#10691 Add depots to airports that have a hangar.
 
+	SLV_DEPOTID_IN_HANGAR_ORDERS,           ///< 320  PR#10691 Go to hangar orders store the DepotID instead of StationID.
```

Comment:
I don't follow why there's two savegame bumps

Reply:
> I don't follow why there's two savegame bumps

**One is for adding the depot structs for airports. (first commit)
The other one is for changing the way goto hangar orders are handled. (second commit)**

If it is better to have only one savegame bump, I can change it, but I was told in the past to keep individual commits with its corresponding savegame bump.

The values XXX and 320, of course, would need to be updated, but I will do it when the PR were to be accepted. I have rebased it so many times, I got tired to do it at some point.

Just let me know what is best in this case.

Reply:
My recollection of past conversations included the idea that each commit should be able to stand alone. An easy way to avoid this debate is to squash when merging. 🙂 

(Curious what other devs think)